SwiftDiffusion/PromptView/PromptTextEditor/PromptEditorView.swift

```diff
@@ -39,6 +42,25 @@ struct PromptEditorView: View {
         )
         .focused($isFocused)
         .animation(.easeInOut(duration: 0.15), value: isFocused)
+      
+      if !loraModelsManager.loraModels.isEmpty && isFocused {
+        HStack {
+          HalfMaxWidthView {}
+          
+          Menu {
+            ForEach(loraModelsManager.loraModels, id: \.id) { lora in
+              Button(lora.name) {
+                let loraSyntax = "<lora:\(lora.alias):1>"
+                text += (text.isEmpty ? "" : " ") + loraSyntax
+              }
+            }
+          } label: {
+            Label("Add LoRA", systemImage: "plus.circle")
+          }
+        }
+        .frame(height: 30)
+      }
```

Comment:
TextEditor is currently animating resize. Static menu appearance might look out of place without similar animation.